tests/fixtures_server.py

```diff
@@ -33,19 +34,25 @@
 from fractal_server.syringe import Inject
 
 
-def get_patched_settings():
+def get_patched_settings(temp_path: Path):
     settings = Settings()
     settings.JWT_SECRET_KEY = "secret_key"
     settings.DEPLOYMENT_TYPE = "development"
     settings.DB_ENGINE = "sqlite"
-    settings.SQLITE_PATH = "_test.db?mode=memory&cache=shared"
+    settings.SQLITE_PATH = (
```

Comment:
Sorry, I cannot find `temp_path` definition. Is it custom or from pytest?

Reply:
notice that `get_patched_settings(temp_path)` is just a function, not a fixutre. Thus `temp_path` is just a parameter name.

Indeed `tmp_path` is a standard fixture of pytest.